commands/get/templates.go

```diff
@@ -0,0 +1,106 @@
+package get
+
+import (
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/feedhenry/rhm/commands"
+	"github.com/feedhenry/rhm/storage"
+	"github.com/urfave/cli"
+)
+
+// NewGetTemplateCmd forms the basis of getting templates of various types
+func (tc *templatesCmd) Templates() cli.Command {
+	return cli.Command{
+		Name:   "templates",
+		Usage:  "templates",
+		Action: tc.templatesAction,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:        "type",
```

Comment:
As this is an option flag, I wonder should we have a default value of project for this. You can do this by setting the 
```Value``` property on the StringFlag definition